### DISCUSSION TITLE
vTaskGetInfo() error.

### DISCUSSION THREAD

**Discussion by ubaldot** [feilipu/Arduino_FreeRTOS_Library#129] [Q&A] [answered]
I am trying to use [vTaskGetInfo()](https://www.freertos.org/vTaskGetInfo.html), but I get `use of undeclared identifier 'vTaskGetInfo'`. Is there any way for using such a FreeRTOS function (btw I observed that there are also other functions missing)?

**Comment by feilipu** [comment, marked as answer]
A quick search of the source reveals it needs to be configured to be enabled. This is true for many API functions.

`#if ( configUSE_TRACE_FACILITY == 1 )`
